### DISCUSSION TITLE
SSO open source integration with google Oauth

### DISCUSSION THREAD

**Discussion by malnajdi** [gravitational/teleport#33790] [General]
Hey, 

I would like to ask if there is anyway to integrate the oss version with oauth google ? it's the only thing that stops me from using it, and if there isn't, any guide on how to add such feature to it maybe i can contribute it back here ?

thanks.

**Comment by webvictim** [comment]
OIDC integration is a Teleport Enterprise feature, so would require an Enterprise license to use.

**Reply by malnajdi** [reply]
Yes, i have seen that @webvictim but my question is, is there any other way that teleport support auth that i can integrate with google ? like example proxy_auth from my reverse proxy to teleport ? where my proxy_auth will implement OIDC ?

**Reply by webvictim** [reply]
Not that I'm aware of.
